qgreenland_dowload.py

```diff
@@ -607,7 +642,10 @@ def download_data(self):
                     f.write(reply_content)
             
             self.progressBar.setValue(int((current + 1) * total))
-
+        
+        # set the final progress bar text with the amount of megabytes downloaded
+        self.progressBar.setFormat(f"{total_size:,} MB have been downloaded")
```

Comment:
What do you think about `{total_size:,.2f}` to limit the number of decimal places to 2?

Reply:
yes, makes sense